internal/ghworkflow/defaults.go

```diff
@@ -17,13 +17,29 @@ package ghworkflow
 import "strings"
 
 ///////////////////////////////////////////////////////////////////////////////
-// Constants and Variables
+// Constants
 
 const (
 	defaultPostgresVersion = "12"
 	defaultRunnerOS        = "ubuntu-latest"
 )
 
+const (
+	cacheAction            = "actions/cache@v3"
+	checkoutAction         = "actions/checkout@v3"
+	setupGoAction          = "actions/setup-go@v3"
+	dependencyReviewAction = "actions/dependency-review-action@v1"
```

Comment:
The only thing the v3 update changed was the supported node version which is only relevant for self hosted actions runners. 🙄

Reply:
Yeah but we still have to keep the actions up-to-date 🤷🏾‍♂️.